org2elcomment.el

```diff
@@ -78,21 +78,82 @@
 (require 'pulse nil t)
 
 (defvar org2elcomment-backend 'ascii)
-
 (defvar org2elcomment-last-source nil)
 
-(make-variable-buffer-local 'org2elcomment-last-source)
+(defvar org2elcomment-readme "README.org")
 
 (defun org2elcomment--find-bounds (buffer)
   (let (beg end)
     (with-current-buffer buffer
       (save-excursion
         (goto-char (point-min))
-        (when (re-search-forward "^;;; Commentary:$" nil t)
+        (when (re-search-forward "^;;;[[:blank:]]+Commentary:[[:blank:]]*$" nil t)
           (setq beg (line-beginning-position 2))
-          (when (re-search-forward "^;;; Code:$")
+          (when (re-search-forward "^;;;[[:blank:]]+Code:[[:blank:]]*$")
             (setq end (line-beginning-position))
-            (cons beg end)))))))
+            (if (and beg end)
+                (cons beg end)
+              (message "org2elcomment: No \";;; Commentary:\" or \";;; Code:\" found.")
+              nil)))))))
+
+;;;###autoload
+(defun org2elcomment-anywhere (el-file &optional readme-file)
+  (interactive
+   (list
+    (let ((prompt
+           (if org2elcomment-last-source
+               (format "Elisp file (default \"%s\"): "
+                       org2elcomment-last-source)
+             "Elisp file: ")))
+      (setq org2elcomment-last-source
+            (read-file-name prompt
+                            nil
+                            org2elcomment-last-source
+                            t)))))
+  (let* ((el-file-dir (file-name-directory el-file))
+         (readme-file (or readme-file
+                          (when org2elcomment-readme
+                            (concat el-file-dir org2elcomment-readme))
+                          (read-file-name "README.org file: " nil nil t)))
+         output)
+    (unless (file-exists-p readme-file)
+      (setq readme-file (read-file-name "README.org file: " el-file-dir)))
+    (with-temp-buffer
+      (insert-file-contents readme-file)
+      (setq output
+            (concat
+             (format "Note: don't edit the commentary by hand, which is auto generated by `org2elcomment' \n\n")
+             (org-export-as org2elcomment-backend))))
+    (when output
+      (with-temp-buffer
+        (emacs-lisp-mode)
+        (insert-file-contents el-file)
+        (goto-char (point-min))
+        (let ((bounds (org2elcomment--find-bounds (current-buffer)))
+              beg)
+          (when bounds
+            (kill-region (car bounds) (cdr bounds))
+            (goto-char (car bounds))
+            (insert "\n")
+            (setq beg (point))
+            (insert output)
+            (comment-region beg (point))
+            (insert "\n")
+            (setq output (buffer-string))
+            (if (file-locked-p el-file)
+                (message "org2elcomment: File %S has been locked, deny operate it! " el-file)
```

Comment:
Small grammar fix: `File %S has been locked, deny operate it!` to `File %S has been locked. Operation denied.`